### ISSUE TITLE
Error 'transl' on fresh install

### ISSUE THREAD

**Issue by Aergernis** [mauro-midolo/homeassistant_electrolux_status#82]
Hi,
i'm trying to get the addon to work, but after login i just get this error.

```
Dieser Fehler wurde von einer benutzerdefinierten Integration verursacht

Logger: custom_components.electrolux_status
Source: custom_components/electrolux_status/__init__.py:94
Integration: Electrolux Care Integration (documentation, issues)
First occurred: 16:28:14 (1 occurrences)
Last logged: 16:28:14

'transl'
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/pyelectroluxconnect/Session.py", line 613, in _parseApplianceStateItem
    ]["valTransl"] = profileItem[1]["steps"][stepKey]["transl"]
KeyError: 'transl'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/pyelectroluxconnect/Session.py", line 638, in _parseApplianceStateContainer
    self._parseApplianceStateItem(
  File "/usr/local/lib/python3.10/site-packages/pyelectroluxconnect/Session.py", line 622, in _parseApplianceStateItem
    raise Error(err)
pyelectroluxconnect.Session.Error: 'transl'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/pyelectroluxconnect/Session.py", line 572, in _parseApplianceState
    result[_hexHacl]["container"] = self._parseApplianceStateContainer(
  File "/usr/local/lib/python3.10/site-packages/pyelectroluxconnect/Session.py", line 653, in _parseApplianceStateContainer
    raise Error(err)
pyelectroluxconnect.Session.Error: 'transl'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/config/custom_components/electrolux_status/__init__.py", line 94, in _async_update_data
    appliance_state = await self.hass.async_add_executor_job(self.api.getApplianceState, appliance)
  File "/usr/local/lib/python3.10/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/local/lib/python3.10/site-packages/pyelectroluxconnect/Session.py", line 967, in getApplianceState
    return self._parseApplianceState(
  File "/usr/local/lib/python3.10/site-packages/pyelectroluxconnect/Session.py", line 581, in _parseApplianceState
    raise Error(err)
pyelectroluxconnect.Session.Error: 'transl'
```

**Comment by tomeko12** [comment]
Hi.  
what is Your appliance PNC/ELC number (9 digits + 2 additinal digits)? What language have you set?

**Comment by Aergernis** [comment]
916099280 02 and 914501217 00
I've tried with german and english

**Comment by mauro-midolo** [comment]
@Aergernis, the problem should be fixed in v2.6.1
